repo_remote.go

```diff
@@ -285,10 +285,10 @@ type RemoteSetURLDeleteOptions struct {
 	Timeout time.Duration
 }
 
-// RemoteSetURLDelete deletes the remote with given name of the repository in
+// RemoteDeleteURL deletes the remote with given name of the repository in
 // given path.
-func RemoteSetURLDelete(repoPath, name string, opts ...RemoteSetURLDeleteOptions) error {
-	var opt RemoteSetURLDeleteOptions
+func RemoteDeleteURL(repoPath, name string, opts ...RemoteDeleteURLOptions) error {
+	var opt RemoteDeleteURLOptions
```

Comment:
`RemoteSetURLDelete` was used to be aligned with how Git bin is used, as `"remote", "set-url", "--delete"`. If we skip the `"set-url"` part, then I think we are not following how a user uses Git bin anymore, which kinda breaks the naming convention we try to develop since this PR.

Reply:
ok, reverting it